### PR TITLE
refactor(server): e2e

### PR DESCRIPTION
**Pull request by jrasm91**
Man, the person e2e tests are rough. There's not enough functionality exposed through the API at the moment.

### CODE REVIEW

File: e2e/src/api/specs/activity.e2e-spec.ts [hunk 1]
```diff
@@ -245,11 +271,11 @@ describe(`${ActivityController.name} (e2e)`, () => {
     });
 
     it('should return a 200 for a duplicate like on the album', async () => {
-      const reaction = await api.activityApi.create(server, admin.accessToken, {
-        albumId: album.id,
-        type: ReactionType.LIKE,
-      });
-      const { status, body } = await request(server)
+      const [reaction] = await Promise.all([
```
Comment (danieldietzler) [inline e2e/src/api/specs/activity.e2e-spec.ts:274]: What's the reason for the promise.all here and a few lines below? Future-proofing?

File: e2e/src/api/specs/person.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    });
+
+    it('should return only visible people', async () => {
+      const { status, body } = await request(app)
+        .get('/person')
+        .set('Authorization', `Bearer ${admin.accessToken}`);
+
+      expect(status).toBe(200);
+      expect(body).toEqual({
+        total: 2,
```
Comment (danieldietzler) [inline e2e/src/api/specs/person.e2e-spec.ts:73]: Why is that total 2?

**Review by danieldietzler — APPROVED** [review]
Nice! 🎉

File: e2e/src/api/specs/person.e2e-spec.ts (hunk 2 above)
Comment (jrasm91) [reply e2e/src/api/specs/person.e2e-spec.ts:73]: The response has total: number and items: Person[] or something like that. This returns both people.

Comment (danieldietzler) [reply e2e/src/api/specs/person.e2e-spec.ts:73]: Shouldn't it only return the visible person?

Comment (jrasm91) [reply e2e/src/api/specs/person.e2e-spec.ts:73]: Idk. Maybe the test case is mislabelled now. I just copied it is as.

**Pull request merged**